<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_addBoxes_automaticDistribution_distanceBetweenCenter</class>
 <widget class="QDialog" name="Dialog_addBoxes_automaticDistribution_distanceBetweenCenter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>372</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>452</width>
    <height>372</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>452</width>
    <height>372</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Automatic distribution: set distance between box centers</string>
  </property>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>319</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_createBoxes">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>319</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create boxes</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_startingPoint">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>411</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Starting point along the trace</string>
   </property>
   <widget class="QRadioButton" name="radioButton_firstPoint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>31</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>First point (green double circle in ImageView)</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_lastPoint">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>61</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Last point (red double triangle in imageView)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_pixelBoxSize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>411</width>
     <height>95</height>
    </rect>
   </property>
   <property name="title">
    <string>Box size in pixel</string>
   </property>
   <widget class="Widget_stackedProfilBox_editSize" name="widget_stackedProfBox_editSize" native="true">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>20</y>
      <width>301</width>
      <height>74</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_distanceBetweenCenters">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Distance between centers of two successives boxes</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_distanceBetweenCenters">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>35</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_distanceBetweenCenters_unit">
    <property name="geometry">
     <rect>
      <x>235</x>
      <y>36</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>pixels</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget_stackedProfilBox_editSize</class>
   <extends>QWidget</extends>
   <header>widget_stackedprofilbox_editsize.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>widget_stackedProfBox_editSize</tabstop>
  <tabstop>lineEdit_distanceBetweenCenters</tabstop>
  <tabstop>radioButton_firstPoint</tabstop>
  <tabstop>radioButton_lastPoint</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_createBoxes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
